sunpy/map/maputils.py

```diff
@@ -164,11 +166,73 @@ def solar_angular_radius(coordinates):
     return sun._angular_radius(coordinates.rsun, coordinates.observer.radius)
 
 
-def sample_at_coords(smap, coordinates):
+def get_neighboring_indices(indices, position, dimensions):
```

Comment:
This function probably should be private (i.e., add an underscore to the beginning of the name)

Reply:
Helpers are now private.